code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -214,7 +214,7 @@
 
 	if(nutrition < NUTRITION_LEVEL_STARVING - 50)
 		msg += "[t_He] [t_is] severely malnourished.\n"
-	else if(nutrition >= NUTRITION_LEVEL_FAT)
+	else if(nutrition >= NUTRITION_LEVEL_FAT || (HAS_TRAIT(src, TRAIT_VORACIOUS)))
 		if(user.nutrition < NUTRITION_LEVEL_STARVING - 50)
 			msg += "[t_He] [t_is] plump and delicious looking - Like a fat little piggy. A tasty piggy.\n"

```

Comment:
what the FUCK

Reply:
Time to roll back the commits and find out who the fuck made this.

Reply:
It's from ~2012, ancient curse